### PR TITLE
Updates to templates for in tree release tasks.

### PR DESCRIPTION
**Pull request by mozbhearsum**
I _think_ this covers everything except ESR. The beta publishing for RC docs don't exist yet, so they're a TODO before this is mergable.

### CODE REVIEW

**Comment by lundjordan** [comment]
Soon, we will be able to delete wikimo how-tos, this repo's old-how-tos and this repo's Wiki non TC intree how-tos.

I can't wait!

File: releasewarrior/templates/fennec/release.json.tmpl [hunk 1]
```diff
@@ -18,8 +18,8 @@
             },
             {
                 "alias": "pushapk",
-                "description": "run pushapk (out of tree relpro)",
-                "docs": "https://github.com/mozilla-releng/releasewarrior-2.0/blob/master/old-how-tos/relpro.md#3-push-to-releases-dir-mirrors",
+                "description": "run pushapk",
+                "docs": "https://github.com/mozilla-releng/releasewarrior-2.0/wiki/Push-to-Google-Play#what-to-do",
```
Comment (lundjordan) [inline releasewarrior/templates/fennec/release.json.tmpl]: @mozbhearsum actually, didn't you hit issues with this? I thought for <=58.x fennec, we still need to use "publish_fennec" flavour, not the new beta "ship_fennec" flavour.

iow - we do this for current fennec dot releases: https://github.com/mozilla-releng/releasewarrior-2.0/blob/10d9b968cfb1fd812dd3425c77e2638ba29ead4c/old-how-tos/fennec-temp-relpro.md#publish-release

I could be wrong

Comment (mozbhearsum) [reply releasewarrior/templates/fennec/release.json.tmpl]: Yeah, I think you're right. My changes to all of these templates assume we're using in-tree releases everywhere except ESR - so the release/RC changes are equally invalid. Perhaps we should wait until merge day to merge this patch?

**Comment by lundjordan** [comment]
We are going to have to wait until merge day to fix this patch. Though this will need fixing now: https://github.com/mozilla-releng/releasewarrior-2.0/pull/57#discussion_r165218294

**Comment by MihaiTabara** [comment]
@lundjordan @mozbhearsum  What do we do with this PR?

**Comment by mozbhearsum** [comment]
Let's close it. Things have changed a lot, it's not worth unbitrotting. We can open a new one for the next set of updates.

**Pull request closed without merging**